src/Randomizer.SMZ3/FileData/Patches/DiggingGamePatch.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+
+namespace Randomizer.SMZ3.FileData.Patches;
+
+public class DiggingGamePatch : RomPatch
+{
+    public override IEnumerable<(int offset, byte[] data)> GetChanges(PatcherServiceData data)
```

Comment:
Maybe we should turn the tuple into its own type with this many different classes using it now

Reply:
Changed the tuple to a new GeneratedPatch class